source/_integrations/zha.markdown

```diff
@@ -510,6 +510,16 @@ Since all Zigbee Coordinator radio adapters are very sensitive/susceptible to al
   - Place your Zigbee Coordinator away from any Wi-Fi access points and all other sources of WiFi.
   - Wi-Fi frequency ranges can overlap with Zigbee, see the section above on defining Zigbee channel use.
 
+### Zigbee network visualization in ZHA UI
+
+ZHA configuration UI has a window/tab for *Visualization* that provides a visual representation of device links in your Zigbee network topology.
+
+The network visualization can help to get a quick overview in identifying devices that are reporting links with poor connection (i.e. low values). However, be aware that for deeper troubleshooting you will need to look at the ZHA logs instead.
+
+The visualization shows multi-hop connections between your paired devices and their reported cumulative values of Received Signal Strength Indicator (RSSI) and Link Quality Indication (LQI).
+
+Note that the exact method in which these values are reported depends on the Zigbee network stack being used on each device, and LQI values can be modified at each step as the massage propagates through the mesh networking matrix.
```

Comment:
you can drop `being` from `network stack being used`

Reply:
OK, done.